src/System.Net.Requests/tests/AssemblyInfo.cs

```diff
@@ -0,0 +1,7 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Xunit;
+
+[assembly: CollectionBehavior(CollectionBehavior.CollectionPerAssembly, DisableTestParallelization = true, MaxParallelThreads = 1)]
```

Comment:
I just did a small measure in how long it takes to execute in netcoreapp with and without parallelism and this are the results:

Innerloop without parallelism: 2.68s
Innerloop with parallelism: 2.54s

Innerloop and Outerloop without parallelism: 21.66s
Innerloop and Outerloop with parallelism: 21.65s